srcpkgs/brasero/template

```diff
@@ -14,8 +14,9 @@ short_desc="Application to burn CD/DVD for the Gnome Desktop"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-or-later"
 homepage="http://www.gnome.org/projects/brasero"
```

Comment:
This redirects to https://wiki.gnome.org/Apps which is a bit useless, can we use a proper homepage?

Reply:
Yes, it was also http ... ergh